src/main/java/org/victorrobotics/frc/dtlib/sensor/imu/DTIMU.java

```diff
@@ -0,0 +1,38 @@
+package org.victorrobotics.frc.dtlib.sensor.imu;
+
+import edu.wpi.first.util.sendable.Sendable;
+import edu.wpi.first.util.sendable.SendableBuilder;
+
+public interface DTIMU<IMUTYPE> extends Sendable, AutoCloseable {
+    IMUTYPE internal();
+
+    double getYaw();
+
+    double getPitch();
+
+    double getRoll();
+
+    void zeroYaw();
+
+    @Override
+    default void initSendable(SendableBuilder builder) {
+        builder.addDoubleProperty("Roll", this::getRoll, null);
+        builder.addDoubleProperty("Pitch", this::getPitch, null);
+        builder.addDoubleProperty("Yaw", this::getYaw, null);
+        builder.addDoubleArrayProperty("Angular velocity", this::getAngularVelocities, null);
+        builder.addDoubleProperty("Compass", this::getCompassHeading, null);
+        builder.addStringProperty("Firmware", this::getFirmwareVersion, null);
```

Comment:
Should we add the 3 linear acceleration axes?

Reply:
I didn't look at the Pigeon docs long enough to find the accessors for these values. Planned for the future.

Reply:
Found the accessor, which is `getBiasedAccelerometer(short[])`, and it returns values in  2.14 fixed-point notation. Should be trivial to convert to a double, or even a float.